client/src/Button/Button.jsx

```diff
@@ -1,26 +1,42 @@
 import "./Button.scss";
 // encourage proper use of link for link and button for action
-const Button = ({ context, styleModifier="default", action, text, icon, link }) => {
-    if (!action && !link) return null;
-    return (
-        <>
-            {link
-                ? <a className={`${context} c-button c-button--${styleModifier}`}
-                    href={link}>
-                    <span className="c-button__text">
-                        {text}
-                    </span>
-                     {icon ? <span className={`c-button__icon c-button__icon--${icon}`}></span> : ``}
-                </a>
-                : <button className={`${context} c-button c-button--${styleModifier}`} onClick={action} >
-                    <span className="c-button__text">
-                        {text}
-                    </span>
-                    {icon ? <span className={`c-button__icon c-button__icon--${icon}`}></span> : ``}
-                </button>
-            }
-        </>
-    );
-}
+const Button = ({
+  context,
+  styleModifier = "default",
+  action,
+  text,
+  icon,
+  link,
+}) => {
+  if (!action && !link) return null;
```

Comment:
Instead of just returning `null`, should we just set defaults? If this component is going to get used, it could be confusing if they don't set the action and link button and a component doesn't show up.